src/error.rs

```diff
@@ -137,9 +142,24 @@ impl std::fmt::Display for ErrorKind {
             ErrorKind::BadComponent => write!(f, "Bad component"),
             ErrorKind::BadImage => write!(f, "Bad image"),
             ErrorKind::UnexpectedDuplicate => write!(f, "Unexpected duplicate"),
+            ErrorKind::UnexpectedMove => {
+                write!(f, "Unexpected move point, can only occur at start of contour")
+            }
+            ErrorKind::UnexpectedSmooth => {
+                write!(f, "Unexpected smooth attribute on an off-curve point")
```

Comment:
this looks to me be a bit too harsh. Can't you simply ignore that "smooth" flag on off-curve points?

Reply:
But the spec says it's illegal :O

Reply:
oh does it? Well.. ok